### ISSUE TITLE
Simulate Multiple IP Addresses while running a performance test

### ISSUE THREAD

**Issue by malvikach** [a85/POSTMan-Chrome-Extension#12309]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

Raising this based on a user comment on our blog:

"Hi Malvika, one problem we encountered with load testing is even with virtual users, load is coming from one client machine (source IP address). Does postman have a feature to work around this with a dummy client, or does this need to be setup outside of postman? (ie spinning multiple temp servers)"

### Describe the solution you'd like

Simulate multiple ip addresses while sending requests on performance tests.

**Comment by levontumanyan** [comment]
are there any updates on this?

**Comment by untimated** [comment]
yeah important feature, load testing from different IP may allow us to simulate real scenario environment equipped with rate-limit or policies.

**Comment by saialekhya-001** [comment]
Partial help here: cloud performance runs (beta) distribute the test across multiple machines in Postman's managed infrastructure rather than generating all load from a single client. That removes the single-machine bottleneck, though note it doesn't yet expose configurable multiple source IPs specifically. Worth trying from the Performance tab in the desktop app if the main concern is load originating from one machine.

<img width="1512" height="543" alt="Image" src="https://github.com/user-attachments/assets/1deb6a81-1e07-43b8-9e9c-bc2ee7f2ce75" />
